sup3r/postprocessing/file_handling.py

```diff
@@ -39,7 +40,9 @@
             'relativehumidity': {'scale_factor': 100.0,
                                  'units': 'percent',
                                  'dtype': 'uint16',
-                                 'chunks': (2000, 500)},
+                                 'chunks': (2000, 500),
+                                 'max': 100,
+                                 'min': 0},
             'pressure': {'scale_factor': 0.1,
                          'units': 'Pa',
```

Comment:
Can you add physical ranges to all of the variables? Except maybe BVF stuff. With ML stuff it's totally worth protecting against spurious outputs. The ranges can be very generous. For example: 

https://github.com/NREL/rex/blob/5f15a1fc7ee738716425b52fdb87757f34d24881/rex/sam_resource.py#L56-L77

Reply:
Yeah sure thing